tools/fetch.py

```diff
@@ -156,9 +159,9 @@ def vx_start(nodelist,outdir,cmd, timeout=0, threadcount=20):
     parser.add_option("", "--external", dest="external",  action="store_true",
                         help=("Run commands external to the server. The "+
                               "default is internal."))
-    parser.add_option("", "--template", dest="template", 
-                        help=("Command template for external commands; "+
-                              "substitutes [] with hostname."))
+    parser.add_option(
+        "", "--user", dest="user", metavar="username", default="site_admin",
```

Comment:
Very minor detail: you seem to set default option values for all other arguments in parser.set_defaults(), but this one in parser.add_option().  Would it be more consistent to set the default user in parser.set_defaults() too instead of here?